Cargo.toml

```diff
@@ -484,4 +484,4 @@ webbrowser = "1.0.6"
 [patch.crates-io]
 bincode = { git = "https://github.com/bgw/bincode.git", branch = "bgw/patches" }
 virtue = { git = "https://github.com/bgw/virtue.git", branch = "bgw/fix-generic-default-parsing" }
-mdxjs = { git = "https://github.com/mischnic/mdxjs-rs.git", branch = "swc-core-32" }
+mdxjs = { git = "https://github.com/vercel-labs/mdxjs-rs-turbopack.git", branch = "turbopack" }
```

Comment:
I moved the fork here: https://github.com/vercel-labs/mdxjs-rs-turbopack

That way, anyone on the team can commit to the repository.